### ISSUE TITLE
Loadscreen improvements

### ISSUE THREAD

**Issue by sprunk** [ZeroK-RTS/Zero-K#2596]
Loadscreens are somewhat outdated:
 - [ ] [command queues](https://github.com/ZeroK-RTS/Zero-K/blob/master/bitmaps/loadpictures/commandqueueing.png): very prominent soon-to-be-obsolete tankfac model. Might also be worth mentioning that factory queues work a bit differently (ALT instead of SPACE for inserting, Repeat applies to builds not orders).
 - [ ] [visibility overlays](https://github.com/ZeroK-RTS/Zero-K/blob/master/bitmaps/loadpictures/fkeys.png): see #2595 
 - [ ] [unit guide](https://github.com/ZeroK-RTS/Zero-K/blob/master/bitmaps/loadpictures/unitguidewarrior.png): dead unitguide link (should link to mediawiki)
 - [ ] [hotkeys](https://github.com/ZeroK-RTS/Zero-K/blob/master/bitmaps/loadpictures/usefulshortcuts.png): idle builder is now `Z` and that sentence shouldn't say "you don't need fancy widgets". The `Ctrl+A` sentence probably shouldn't mention `Ctrl+D` or at least say that it results in a resign. The link at the bottom currently contains very advanced stuff (`uikeys.txt`) so probably shouldn't exist.

**Comment by Licho1** [comment]
Maybe they can be edited.. check artwork for sources for these images. Background selection for them is nice imo.

**Comment by GoogleFrog** [comment]
@The-Yak says that they are able to fulfill requests. We should make a list of tips to add.

**Comment by The-Yak** [comment]
Should the darkhive-style panels be replaced with new techpanel ones? Is there a source of the current techpanels theme?

**Comment by GoogleFrog** [comment]
Perhaps it is worth updating the loadscreen borders. I don't know how important the consistency is here. Here are the images used for the current UI: https://github.com/ZeroK-RTS/Chobby/tree/master/LuaMenu/widgets/chili/skins/Evolved

I think the source are somewhere.

**Comment by The-Yak** [comment]
Thanks, the psp files wont work for me but I should be able to make do with the pngs. Is there an up to date list of hotkeys somewhere?

**Comment by sprunk** [comment]
[All idle are still `Ctrl+B`.](https://github.com/ZeroK-RTS/Zero-K/blob/master/LuaUI/Configs/zk_keys.lua#L165) That file contains all ZK hotkeys but there might be some extra Spring builtins not listed there.

**Comment by GoogleFrog** [comment]
The file may not contain all hotkeys. Widgets can add default hotkeys. The safest way is to get a fresh install.

**Comment by sprunk** [comment]
I would leave F11 out, there's already a prominent button for it which also says the hotkey.

**Comment by The-Yak** [comment]
I still don't understand how to make circle guard work, sorry.

**Comment by sprunk** [comment]
Yes, but note that if the unit moves away it won't work (it has to be in the center when you stop dragging, not when you start).
